### PR TITLE
Mm qna init2

### PR DESCRIPTION
**Pull request by morahan**
initialize my Q&A files & structure.  minor change to App.jsx (added header as placeholder). two minor changes to index.jsx to solve errors on my end (compose was causing issues for me).

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -25,9 +25,10 @@ class App extends Component {
     return (
       <div className="App">
         <div className="container-fluid">
+          <div className="header">Header</div>
```
Comment (shawnsull3) [inline src/App.jsx:28]: What's the reasoning for the header?

File: src/index.jsx [hunk 2]
```diff
@@ -24,10 +22,7 @@ const reducer = combineReducers({
 
 const middleware = [thunk];
 /* eslint-disable no-underscore-dangle */
-const store = createStore(reducer, compose(
-  applyMiddleware(...middleware),
-  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
-));
```
Comment (shawnsull3) [inline src/index.jsx:30]: @thomHayner @jwhooperjr Mike and I determined that this was what was causing the strange errors Mike was seeing. We should be able to use the react devtools to help debug redux stuff as well

**Review by shawnsull3 — APPROVED** [review]
Looks good to me, just have the question about why the header was added in App.jsx.

File: src/App.jsx (hunk 1 above)
Comment (morahan) [reply src/App.jsx:28]: Not sure if header is outlined in Product overview. It seems out of scope of product overview so I figured that we still need a small space for it at the top (logo left & search right). Maybe I am wrong and it is in your row. Lmk

Comment (shawnsull3) [reply src/App.jsx:28]: Yeah I know what you're talking about now. There was anything in the product overview section about, but once I finish my shit I can take care of that

**Pull request merged**